src/backend/distributed/shared_library_init.c

```diff
@@ -701,6 +701,19 @@ RegisterCitusConfigVariables(void)
 		GUC_NO_SHOW_ALL,
 		NoticeIfSubqueryPushdownEnabled, NULL, NULL);
 
+	DefineCustomIntVariable(
```

Comment:
shall we add a regression test, maybe just copy & past these: https://github.com/citusdata/citus/blob/56245d232d8f0f053b4ad658bc57a2a4c055f272/src/test/regress/sql/single_node.sql#L898

with set citus.enable_local_execution TO false?